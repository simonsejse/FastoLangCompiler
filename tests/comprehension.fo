fun int write_int(int x) = write(x)
fun [int] write_int_arr([int] x) = map(write_int, x)

fun [int] main() =
  let n = read (int) in
  let x = iota (n)   in
  let y = iota (n)   in
  write_int_arr ([int r*r | i <- x, j <-y; int r = (i+j) / 2; r - (r/5)*5 == 0 ])
