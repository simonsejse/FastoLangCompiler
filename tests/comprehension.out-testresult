Parse error: Error at line 8, column 18
