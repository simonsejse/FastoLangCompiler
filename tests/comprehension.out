0 0 25 25 0 25 25 25 25 25 25 25 25 25 25 25 25 100 25 25 100 100 25 25 100 100 25 25 100 100 25 25 100 100 25 100 100 100 100 100 100 100 100
